src/Tools/GenerateAnalyzerNuspec/Program.cs

```diff
@@ -265,7 +265,6 @@
     }
 }
 
-result.AppendLine(FileElement(Path.Combine(assetsDir, "EULA.rtf"), ""));
```

Comment:
I haven't removed the file itself because it is still referenced by the `.vsix` files.

Reply:
Any reason to not remove all its references and delete it?

Reply:
It's still used by vsix and as we keep generating them, I don't know what the team wants to do. But that's why I did called it out.